.renovaterc.json

```diff
@@ -1,26 +1,3 @@
 {
-  "extends": ["github>adobe/helix-shared"],
-  "ignoreDeps": [
-    "@spectrum-web-components/action-button",
```

Comment:
Wondering if removing this could put us in danger for things breaking by renovate in the future.

Reply:
What is the reason for removing?

Reply:
The reason we added was because we were very much locked in to a specific older version of swc before this PR. Now that we are on the latest in theory we could allow renovate to keep us up to date.